lib/samples/add_feature_layers/add_feature_layers_sample.dart

```diff
@@ -30,28 +33,45 @@ class AddFeatureLayersSample extends StatefulWidget {
 }
 
 class _AddFeatureLayersSampleState extends State<AddFeatureLayersSample> {
-  final _mapViewController = ArcGISMapView.createController();
+  // create a map with a topographic basemap style.
   final _map = ArcGISMap.withBasemapStyle(BasemapStyle.arcGISTopographic);
+  // create a map view conroller.
+  final _mapViewController = ArcGISMapView.createController();
+  // create a list of feature layer sources.
   final _featureLayerSources =
-      List<DropdownMenuItem<String>>.empty(growable: true);
-  String? _selectedFeatureLayerSource;
+      List<DropdownMenuItem<Source>>.empty(growable: true);
```

Comment:
Is there a reason we have to use `.empty(growable: true)`? Why not just

```dart
final _featureLayerSources = <DropdownMenuItem<Source>>[];
```

Reply:
You are right it works without it! It was there before I started wroking on it so I didn't touch it